pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -61,7 +54,8 @@ task qiime_import_from_bam {
         docker: docker
         memory: "${memory_mb} MiB"
         cpu: cpu
-        disks: "local-disk ${disk_size_gb} SSD"
+        disk: disk_size_gb + "GB"
```

Comment:
I'm pretty sure TES wants a space between the number and the unit (would want to apply this change to all tasks)
```suggestion
        disk: disk_size_gb + " GB"
```

Reply:
Made edits across all script